src/main/resources/webapp/interface/index.ts

```diff
@@ -0,0 +1,2 @@
+export {list} from './list';
```

Comment:
Cant these two lines be included in the spots they are used? So we don't need a seperate file for this?

Reply:
No, this is common and good practice.